src/components/editor/editor-pane/editor-pane.tsx

```diff
@@ -157,16 +208,25 @@ export const EditorPane: React.FC<EditorPaneProps & ScrollProps> = ({ onContentC
   }), [t, editorPreferences])
 
   return (
-    <div className={'d-flex flex-column h-100'} onMouseEnter={onMakeScrollSource}>
+    <div className={'d-flex flex-column h-100 position-relative'} onMouseEnter={onMakeScrollSource}>
       <MaxLengthWarningModal show={showMaxLengthWarning} onHide={() => setShowMaxLengthWarning(false)} maxLength={maxLength}/>
       <ToolBar
         editor={editor}
       />
+      <ShowIf condition={showDropOverlay}>
+        <DropOverlay
+          onDrop={onDropCallback}
+          onDragLeave={() => setShowDropOverlay(false)}
```

Comment:
Even if this function is short, please use a callback here. Every time this component gets rerendered, the function will be recreated. React will see this as a change of the prop and will rerender the drop overlay.
That's why you should useCallbacks in components that rerender a lot.

Reply:
done

Reply:
I won't resolve this, because this is not solved. There are still inline callbacks.

Reply:
I fixed the one other thing I saw. If you have anything else please specify it

Reply:
Line 226

Reply:
Well true, but that's not something I did… Should this be fixed in this PR?